mongodb:
  host: 'localhost'
  port: 62345
vision-project:
  controllers:
    is_debug: False
    update_window_seconds: 15
    minutes_between_interactions: 1
    scheduled_window_minutes: 15
  params:
    seconds_until_interaction_timeout: 30
    max_num_of_prompted_per_day: 3
  data:
      evaluation: '/root/evaluation'
      data_capture: '/root/upload'
